ffc/parameters.py

```diff
@@ -52,7 +55,7 @@
     "external_include_dirs": "",
 }
 _FFC_CACHE_PARAMETERS = {
-    "cache_dir": "",  # cache dir used by Instant
+    "cache_dir": "compile_cache",  # cache dir used by default
```

Comment:
Use `pathlib` to make clear that the strings are actually filesystem directory names.